clueless/static/js/controllers.js

```diff
@@ -10,7 +10,7 @@ class GameState {
     this.witnessItems = [null, null, null];
     this.gameid = null;
     this.lastSuggestion = null;
-    this.registered = [];
```

Comment:
why is `registered` being deleted here?

Reply:
Doesn't that undo/conflict with #47 ?

Reply:
Registered was actually never used - instead, it just instantiated the players that were already registered in `gameState.players`. After searching to see if it was used anywhere and coming up with nothing, I took it out.